test/corpus/declarations.txt

```diff
@@ -436,6 +436,7 @@ class Point {
   Point(int x, int y) {
     this.x = x;
     this.y = y;
+    this.ABCÃ¢ = new @Anno Point();
```

Comment:
Again would be better to separate and have a section for Annotation on constructor.
Here you added a test about annotations; it has nothing to do really with the title of test which is "constructor declaration"